ext/standard/tests/array/array_fill_error2.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+Test array_fill() function : error conditions - count is too large
+--FILE--
+<?php
+echo "*** Testing array_fill() : error conditions - count is too large ***\n";

```

Comment:
This echo isn't very helpful.

Reply:
To clarify, I would just remove this completely :slightly_smiling_face:

Reply:
Ok I misunderstood and thought I needed to adapt it to follow the same logic as other error test files, thanks for clarifying.